program-runtime/src/sysvar_cache.rs

```diff
@@ -26,189 +26,191 @@ impl ::solana_frozen_abi::abi_example::AbiExample for SysvarCache {
 
 #[derive(Default, Clone, Debug)]
 pub struct SysvarCache {
```

Comment:
Do we have a measurement of the actual size allocated across the entire struct here? Because now it is no longer shared between banks but cloned, right?

Reply:
i dug into this last month when discussing `SysvarCache` with firedancer while prepping the simd and as best i can tell the object is created fresh for every bank

`_new_from_parent` sets `transaction_processor: TransactionBatchProcessor::default(),` when it instantiates a new bank, and the processer contains the `SysvarCache`. later, `_new_from_parent` updates the account data for the sysvars, then near the end calls `transaction_processor.fill_missing_sysvar_cache_entries` which routes to `sysvar_cache.fill_missing_entries`, which loads the account data using a callback via `banks_client` and assigns it to `SysvarCache`

the other reason to be quite sure `SysvarCache` is never shared or reused is because the setter functions only do the assignment if the entry is `None`. in other words if the cache _were_ reused in any way, the clock wouldnt tick